R/utilities-enums.R

```diff
@@ -1,3 +1,84 @@
+#' @title .getTitlesFromFamilyTag
+#' @description Get all title names from documented functions/R6 classes that have specific family tags.
+#' The function aims at created automated and synchronized enums
+#' @param familyTag Family tag used to document functions or R6 classes
+#' @return character array of titles
+#' @import ospsuite.utils
+#' @keywords internal
+.getTitlesFromFamilyTag <- function(familyTag) {
+  functionNames <- NULL
+  for (filePath in list.files("./R", full.names = TRUE)) {
```

Comment:
no idea how it works - but it works :)

Reply:
When building/documenting a package, the working directory is internally set to the package location during the procedure (I did some testing with `devtools::check()` and `devtools::document()`).

The function `.getTitlesFromFamilyTag` is only called during the build of the package to create the enums.
Thus, when the users check for e.g. `Atoms`, they do not call for `.getTitlesFromFamilyTag` but directly for the resulting list created during the package construction.

Reply:
ok, cool. Thanks for the explanation :)